api/src/main/java/org/openmrs/api/db/hibernate/HibernateConditionDAO.java

```diff
@@ -103,6 +103,19 @@ public Condition getConditionByUuid(String uuid) {
 		query.setInteger("patientId", patient.getId());
 		return query.list();
 	}
+
+	/**
+	 * @see ConditionService#getAllConditions(Patient)
+	 * @since 2.4.0
```

Comment:
The since annotation should be only on the service interface, ConditionService

Reply:
@dkayiwa  I updated the PR.